Frontend/src/pages/admin/AdminCreateNotificationPage.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <BasePage title="Send Notification">
+    <q-card>
+      <div class="q-pa-md">
+        <q-table
+          title="Select Students"
+          :data="tableStudents"
+          :columns="columns"
+          row-key="email"
+          selection="multiple"
+          :selected.sync="toSendTo"
+        />
+        <q-input
+          outlined
+          v-model="title"
+          label="Title"
+          class="notifPageComponents"
+        />
+        <q-input
+          v-model="body"
+          label="Body"
+          outlined
+          type="textarea"
+          class="notifPageComponents"
+        />
+        <q-btn
+          color="primary"
+          @click="sendNotification()"
+          label="Send Notification"
+          class="notifPageComponents"
+        />
+      </div>
+    </q-card>
+  </BasePage>
+</template>
+
+<script>
+import BasePage from "../BasePage.vue";
+
+export default {
+  components: {
+    BasePage
+  },
+  props: {
+    selected: {
+      type: Array,
+      required: false
+    }
+  },
+  name: "HomeMainInfo",
+
+  data: () => ({
+    tableStudents: [],
+    toSendTo: [],
+    body: "",
+    title: "",
+    selectedString: "Please select a student",
+    columns: [
+      {
+        name: "studentLastName",
+        required: true,
+        label: "Student Last Name",
+        align: "left",
+        field: "lastName",
+        sortable: true,
+        classes: "my-class",
+        style: "width: 500px"
+      },
+      {
+        name: "studentFirstName",
+        required: true,
+        label: "Student First Name",
+        align: "left",
+        field: "firstName",
+        sortable: true,
+        classes: "my-class",
+        style: "width: 500px"
+      },
+      {
+        name: "studentID",
+        required: true,
+        label: "Student ID",
+        align: "left",
+        field: "id",
+        sortable: true,
+        classes: "my-class",
+        style: "width: 500px"
+      },
+      {
+        name: "email",
+        required: true,
+        label: "Email",
+        align: "left",
+        field: "email",
+        sortable: true,
+        classes: "my-class",
+        style: "width: 500px"
+      }
+    ]
+  }),
+  created: function() {
+    this.$axios
+      .get("/students", {
+        headers: {
+          Authorization: this.$store.state.token
+        }
+      })
+      .then(resp => {
+        this.tableStudents = resp.data;
+      });
+    this.toSendTo = this.selected;
+  },
+  methods: {
+    goToStudentCoop() {
+      this.$router.push("/admin/studentcoops");
```

Comment:
nit: change this route to `/admin/student-coops`

Reply:
this isnt even a real page but ok lol